### ISSUE TITLE
Add links to exotenportaal

### ISSUE THREAD

**Issue by damianooldoni** [inbo/craywatch#1]
We would like to add links to GAM and other interesting information about IAS crayfishes species.
Has each species in the exotenportaal a unique URL? @timadriaens says "yes, I am quite sure of it".

@sanderdevisscher: can you provide us the links to the crayfishes here below:
- Faxonius limosus
- Faxonius rusticus
- Faxonius virilis
- Pacifastacus leniusculus
- Procambarus clarkii
- Procambarus virginalis

**Comment by SanderDevisscher** [comment]
@damianooldoni currently the exotenportaal app is still in the development phase. But we intend to indeed enable direct linking to the species information page of a species.  
Currently these links are encoded as such (example _Pacifastacus leniusculus_):
"http://alienspecies.inbo.be/?page=species_information&language=en&taxonkey=2226990&gewest=flanders,wallonia,brussels"
This is a temporary link format since the PRD domain hasn't been assigned yet.

Also I noticed some of these species are currently missing 🤔 I expect taxonkey mismatches & non up to date data flows to be the culprit. I'll investigate.

**Comment by SanderDevisscher** [comment]
@damianooldoni I did a quick look at the source and I noticed the following issues:

| Species               | Issue                                   |
|-------------------------|-----------------------------------------|
| Faxonius  limosus    | on GRIIS but no data on alienspecie portal while 4164 on gbif |
| Faxonius  rusticus   | not on GRIIS                            |
| Faxonius  virilis    | not on GRIIS                            |
| Procambarus  virginalis | not on GRIIS                            |

W'll need to check whether Faxonius limosus is included in the Zenodo alienspecies cube. 
Also some more investigation into why the other species are missing from the GRIIS checklist is needed.

**Comment by damianooldoni** [comment]
Thanks for the update!

**Comment by fjasteen** [comment]
A bit late as an answer, but up to now, there is no F. virilis, nor F. rusticus in Belgium, hence their missing? As for virginalis, first record is 2020

**Comment by timadriaens** [comment]
indeed, that is the reason. their non-occurrence is probably indeed the classic taxonomic issue (P. virginalis was formerly Procambarus fallax virginalis, F. limosus used to be Orconectes limosus). But I think meanwile we fixed this with the gbif publication of the Union List @damianooldoni ?
